codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ComponentSourceGenerator.scala

```diff
@@ -117,8 +119,8 @@ object ComponentSourceGenerator {
       val methods = component.callableCommands
         .map { command =>
           val commandMethod = lowerFirst(command.name)
-          val inputType = fullyQualifiedMessage(command.inputType)
-          val outputType = fullyQualifiedMessage(command.outputType)
+          val inputType = fullyQualifiedClassWithRoot(command.inputType)
+          val outputType = fullyQualifiedClassWithRoot(command.outputType)
```

Comment:
Sad that we can't be more 'minimal' here, but reliably detecting when that's needed and when it isn't will be tricky... :+1: for now since this is a 'managed' class anyway.